protected/views/practice/_form.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/* @var $this PracticeController */
+/* @var $model Contact */
+/* @var $form CActiveForm */
+?>
+<?php
+$countries = CHtml::listData(Country::model()->findAll(), 'id', 'name');
+$address_type_ids = CHtml::listData(AddressType::model()->findAll(), 'id', 'name');
+?>
+
+<div class="form">
+
+    <?php $form = $this->beginWidget('CActiveForm', array(
+        'id' => 'practice-form',
+        // Please note: When you enable ajax validation, make sure the corresponding
+        // controller action is handling ajax validation correctly.
+        // There is a call to performAjaxValidation() commented in generated controller code.
+        // See class documentation of CActiveForm for details on this.
+        'enableAjaxValidation' => true,
+    )); ?>
+
+  <p class="note text-right">Fields with <span class="required">*</span> are required.</p>
+    <?php echo $form->errorSummary($model); ?>
+    <div class="row field-row">
+        <div class="large-6 column">
+            <div class="row field-row">
+                <div class="large-3 column"><?php echo $form->labelEx($contact, 'title'); ?></div>
+                <div class="large-4 column end">
+                    <?php echo $form->telField($contact, 'title', array('size' => 15, 'maxlength' => 20)); ?>
```

Comment:
This should use textField and not telField. Remove/replace the length values as well.